conf/solr-biblio/works/conf/schema.xml

```diff
@@ -450,6 +450,7 @@
     <field name="has_fulltext" type="boolean"/>
     <field name="title" type="text" required="true"/>
     <field name="title_suggest" type="textgen" required="true"/>
+    <field name="title_facet" type="string" stored="false" indexed="true" multiValued="false"/>
```

Comment:
I don't think this wants to be `string` but instead something like `textgen` or one of the normalized forms (e.g. lowercase unicode NFKC).  Is there a reason not to just index the `title_suggest` field instead of using a new field?